src/main/java/org/netcracker/educationcenter/elasticsearch/Connection.java

```diff
@@ -0,0 +1,51 @@
+package org.netcracker.educationcenter.elasticsearch;
+
+import org.apache.http.HttpHost;
+import org.elasticsearch.client.RestClient;
+import org.elasticsearch.client.RestHighLevelClient;
+
+import java.io.IOException;
+
+/**
+ * This class establishes a connection with Elasticsearch Database using rest client
+ * to perform requests based on the provided builder.
+ *
+ * @author Mikhail Savin
+ */
+public class Connection {
+
+    /**
+     * High-level client instance to establish connection
+     */
+    private RestHighLevelClient restHighLevelClient;
+
+    /**
+     * @return High-level client instance
+     */
+    public RestHighLevelClient getRestHighLevelClient() {
+        return restHighLevelClient;
+    }
+
+    /**
+     * High-level client internally creates the low-level client
+     * (which maintains a pool of connections and starts some threads) to perform requests.
+     * Note that we need both ports to be added via builder.
+     */
+    public synchronized void makeConnection() {
+        if (restHighLevelClient == null) {
+            restHighLevelClient = new RestHighLevelClient(
+                    RestClient.builder(
+                            new HttpHost("142.93.122.167", 9200, "http"),
+                            new HttpHost("142.93.122.167", 9201, "http")));
```

Comment:
Move to app properties

Reply:
```suggestion
String hostname = property.getProperty("hostname");
String scheme = property.getProperty("scheme");
int port1 = Integer.getInteger(property.getProperty("port1"));
int port2 = Integer.getInteger(property.getProperty("port2"));

if (restHighLevelClient == null) {
        restHighLevelClient = new RestHighLevelClient(
                RestClient.builder(
                        new HttpHost(hostname, port1, scheme),
                        new HttpHost(hostname, port2, scheme)));
                        }
```

Is this OK or should I make these parameters class fields?

Reply:
I defined these parameters as private fields of the class with no getters or setters because their values are taken from .properties file using Properties class. Is it good approach or not?